terraform/gcp/envs/dev.tfvars

```diff
@@ -9,6 +9,12 @@ environment = "dev"
 portals = ["demo", "atcp", "ourhealth", "hearthive", "rgp", "cmi"]
 k8s_namespace = "juniper-dev"
 
+artifact_registry = "juniper"
+artifact_registry_project = "broad-juniper-eng-infra"
+artifact_registry_location = "us-central1"
```

Comment:
do these need to be explicitly set here since they're the defaults in variables.tf?